src/rewrite/vct/rewrite/InlineApplicables.scala

```diff
@@ -261,6 +261,14 @@ case class InlineApplicables[Pre <: Generation]() extends Rewriter[Pre] with Laz
     case Perm(loc @ InstancePredicateLocation(pred, obj, args), WritePerm()) if pred.decl.inline =>
       dispatch(InstancePredicateApply(obj, pred, args, WritePerm()(loc.o))(loc.o))
 
+    case Perm(InLinePatternLocation(loc @ PredicateLocation(pred, args), pat), WritePerm()) if pred.decl.inline =>
+      dispatch(PredicateApply(pred, args, WritePerm()(loc.o))(loc.o))
+//      TODO: Or should I do:
+//      dispatch(InlinePattern(PredicateApply(pred, args, WritePerm()(loc.o))(loc.o))(loc.o))
```

Comment:
So this one I wasn't sure about, what do you think?

Reply:
I think the commented alternative is the least surprising option.